### PR TITLE
CW-224 Living style guide

### PR DESCRIPTION
**Pull request by glebme**
# Change

_Created a living style guide and implemented it on the landing page._

## Change reason

There was a need to create cohesion between components that were being created. A living style guide meant that there is guidance now on how to build these components, where to look for styles and how to translate design into code. Continue adding to the SCSS and documenting it to update the living style guide for other members this year and for future years to work on.

## Comments
- This is in no way complete. There are other re-factorings that need to occur but this a crucial first step. Please continue to add to the SCSS as you see fit. There have also been many things that I have spotted as part of implementing the guidelines on the landing page but those can be resolved much more easily because the implementation of the design is more consistent. Also i have not addressed fully the problem of responsive design and hence that will be the next thing that gets updated.
- For reviewers, please only review code that you have written and I have changed. Otherwise also please look at the cohesiveness of the landing page as a whole.
- To get a live version of the stye guide if you are in the `frontend` folder if you run `yarn run kss` and proceed to create a server to serve the files generated in `styleguide`. You can use vscode's live server extension or run a `simplepython` server.

### CODE REVIEW

**Comment by abiramen** [comment]
Does this PR add dependencies?

File: frontend/src/styles/abstracts/_variables.scss [hunk 1]
```diff
@@ -1,39 +1,19 @@
...
-$error-color: #FF0066;
-$warning-color: #FF6600;
-$success-color: #00FF66;
-
-// ----------------------------------------------------------------------
-	// Layout Breakpoints
+// Layout Breakpoints
 // ----------------------------------------------------------------------
 $maxWidth: 1280;
 $laptop-padding:72px;
```
Comment (abiramen) [inline frontend/src/styles/abstracts/_variables.scss:5]: Minor style nit with spacing.

**Comment by abiramen** [comment]
LGTM otherwise.

**Comment by glebme** [comment]
> Does this PR add dependencies?

Yes

File: frontend/src/styles/abstracts/_variables.scss (hunk 1 above)
Comment (glebme) [reply frontend/src/styles/abstracts/_variables.scss:5]: Change back to old spacing?

**Review by paulatennent — COMMENTED** [review]
looks good to me, cant really see anything that should be changed. :)

**Pull request merged**